frontend/src/components/Typography/Subtitle/index.js

```diff
@@ -0,0 +1,12 @@
+import { PropTypes } from "prop-types";
+import styles from "./index.module.css";
+
+const Subtitle = ({ children }) => {
+  return <h2 className={styles.subtitle}>{children}</h2>;
```

Comment:
I'm not sure this should be an H2, what's the difference between this one and Heading2 component?

Reply:
There are some style differences major one being the font family. If not an h2, which element would it be more suitable to use?